atlasdb-api/src/test/java/com/palantir/atlasdb/transaction/api/KeyValueServiceStatusTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.transaction.api;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+public class KeyValueServiceStatusTest {
+    @Test
+    public void allOperationsMeansHealthy() {
+        assertTrue(KeyValueServiceStatus.HEALTHY_ALL_OPERATIONS.isHealthy());
+    }
+    @Test
+    public void mostOperationsMeansHealthy() {
+        assertTrue(KeyValueServiceStatus.HEALTHY_BUT_NO_SCHEMA_MUTATIONS_OR_DELETES.isHealthy());
```

Comment:
It concerning to mark this as healthy, what if an application regularly performs non transactional operations like `truncates` or `deleteRange`, in that case its unhealthy though quorum is available.

It should be upto the application to decide which KVS Status is healthy for it and which is not.

Reply:
Applications that deviate from "normal" usage (by running truncates/deleteRange) can still access the `KeyValueServiceStatus` and use that to decide their health.

However, we don't want all applications to have to write "if Atlas says `HEALTHY` or `HEALTHY_BUT...`, then we're healthy".